src/config/Config.js

```diff
@@ -161,11 +161,10 @@ const Config = {
         'Replace the link of a supported site with its actual title.',
         1
       ],
-      'Link Title in the catalog': [
+      'Link Title in the Catalog': [
```

Comment:
Sorry for the late reply.

The casing is indeed inconsistent with other settings, but a problem with changing the name of a setting is that settings are referenced by name trough 4-chan X(T), so even a case change means `'Link Title in the Catalog'` will be seen as a separate setting from `'Link Title in the catalog'`, so the value an user had set will be reset to the default when updating.

Reply:
Oh alright, I'll change it back.